modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -545,7 +545,7 @@ public Account putAccount(Long accountId, Account account)
 		AccountEntry accountEntry = _accountEntryService.updateAccountEntry(
 			accountId, _getParentAccountId(account), account.getName(),
 			account.getDescription(), _isDeleteLogo(account, null),
-			_getDomains(account), null,
+			_getDomains(account), account.getEmailAddress(),
```

Comment:
attention: with this you're passing the email you receive in the body of the call. If you don't pass any value we are "removing it".
I think here we need to pass always (because we remove the email Address from this layer) the previous value like we do in the patch method